### PR TITLE
Adds felinid language for felinids (and people wearing the cat ears)

### PR DESCRIPTION
**Pull request by ToasterBiome**
# Document the changes in your pull request

Gives felinids their own language. If you do not understand it, you'll get a combination of "meow" "nya" "nyam" "mm" "mrow" and "mmrrrr"

This is good for the game as more languages encourages RP. plus most other "regular" races got them, might as well give some felinid love.

image

dreamseeker_NkcW6ANAeP

you may now also equip the kitty ears and they will grant you the language too you're welcome

# Wiki Documentation

felinids get their own language, should be on the race page

# Changelog

:cl:  
rscadd: gives felinids their own language  
/:cl:

### CODE REVIEW

**Comment by N3D6** [comment]
Literally the best change

**Comment by Reed0506** [comment]
ok so basically donator chat

**Comment by adamsong** [comment]
Humans already have japanese

**Comment by Anvilman6** [comment]
I'm hesitant on this only because it's a change to a donor race, and because it seems like the server can't decide if Felinids are Human or non-Human.

Not a bad idea, I think, but it's a thorny subject.

EDIT: Maybe give Humans and Felinids joint access to Japanese (or an alternative)? Wouldn't want to discount the work you've done here, but maybe that would keep more in line with them being the same but slightly different.

**Comment by nmajask** [comment]
Thought I would bring up [TG's version of a felinid language](https://github.com/tgstation/tgstation/pull/52958), as many of the things brought up in that may also apply here too.

**Comment by ToasterBiome** [comment]
> EDIT: Maybe give Humans and Felinids joint access to Japanese (or an alternative)? Wouldn't want to discount the work you've done here, but maybe that would keep more in line with them being the same but slightly different.

im not gonna pretend that japanese has any standing as a language besides ninja and "fuck we dont have a language".

you wouldn't understand the pain of being a snowflake race without their own language as a l*zard knowing draconic, i'm sorry. 

> Thought I would bring up [TG's version of a felinid language](https://github.com/tgstation/tgstation/pull/52958), as many of the thing brought up in that may also apply here too.

The discussion here is mirrored from the PR basically. It didn't implode TG and it wont implode yogs. Plus, the tg one is weird and actually fetishy yet it still got in. I asked for the FETISH CONTENT tag and was granted it as a joke.

**Comment by truz** [comment]
> Thought I would bring up [TG's version of a felinid language](https://github.com/tgstation/tgstation/pull/52958), as many of the things brought up in that may also apply here too.

Yeah but in the end it still got merged for reasons of making the race more unique. Metagaming using language or whatever doesn't really apply if every other race has their own language aswell.
I don't think this PR is that bad it's not even a weird version of weaboo japanese like the tg felinid language is

File: code/modules/language/felinid.dm
Comment (adamsong) [inline code/modules/language/felinid.dm]: wat

Comment (ToasterBiome) [reply code/modules/language/felinid.dm]: fetish content, lore team please fix

Comment (ToasterBiome) [inline code/modules/language/felinid.dm]:
```suggestion
	desc = "A language devised by felinids to secretly plot against regular humans"
```

Comment (ToasterBiome) [reply code/modules/language/felinid.dm]: is this more your style @adamsong

Comment (adamsong) [reply code/modules/language/felinid.dm]: Yes

**Comment by Cartlord** [comment]
Against this on the grounds of it being a Donor-only language, being a donor is not supposed to grant any gameplay advantage

**Comment by ToasterBiome** [comment]
> Against this on the grounds of it being a Donor-only language, being a donor is not supposed to grant any gameplay advantage

you may now also understand and speak the cat language by equipping kitty ears

**Comment by N3D6** [comment]
Yeah it's a "donor-only" language but it can also be spoken by the curator and pais, why does literally EVERY other race get a languages but felinids don't

**Review by SomeguyManperson — APPROVED** [review]
:^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

**Comment by Anvilman6** [comment]
> Yeah it's a "donor-only" language but it can also be spoken by the curator and pais, why does literally EVERY other race get a languages but felinids don't

Because the server still can't decide if Felinids are their own species or not, and because they're a donor-locked gameplay option. :     ^ )

**Comment by truz** [comment]
> Because the server still can't decide if Felinids are their own species or not, and because they're a donor-locked gameplay option. : ^ )

Last time donators got asked which they'd like more the majority voted for them to be their own species, even if it brings gameplay changes such as command roles they will be excluded from.

**Comment by Anvilman6** [comment]
> Last time donators got asked which they'd like more the majority voted for them to be their own species, even if it brings gameplay changes such as command roles they will be excluded from.

I'm aware, but that poll is effectively meaningless until Staff / Council makes an official stance on it. 

Ultimately I guess a language isn't the biggest or most egregious change to push, HOWEVER, if Felinids are going to start having benefits or actual identity, then that in my view should be _officially_ greenlit first, rather than implemented as small instances of what could be viewed as powercreep for a specifically donation-locked facet of gameplay.

**Comment by Marmio64** [comment]
ah yes, giving more species snowflake languages that they use so they dont have to RP with other people will totally increase rp 
also this is even worse since like everyone else said, they're a donor race without any real downsides, so donors basically get to be human but with a free language/donorspeak too.

**Comment by ToasterBiome** [comment]
> ah yes, giving more species snowflake languages that they use so they dont have to RP with other people will totally increase rp
> also this is even worse since like everyone else said, they're a donor race without any real downsides, so donors basically get to be human but with a free language/donorspeak too.

you may equip the kitty ears to talk in the language too, so non-donors may interact with felinid speaking players

**Comment by KillerOrcaCora** [comment]
All around based PR don't really see the "Donor-language" argument holding up if all you have to do is get cat ears like you really want to talk to a felind anyhow.

**Comment by maxion12345** [comment]
ok now give humans a race specific language

also isn't the point of felinids to be as close to humans as possible

**Comment by ToasterBiome** [comment]
> ok now give humans a race specific language
> 
> also isn't the point of felinids to be as close to humans as possible

https://github.com/yogstation13/Yogstation/pull/12082#issuecomment-901725878

**Comment by Chubbygummibear** [comment]
I cannot believe this is what you were working on when you asked for my help. I deeply regret aiding you and will proceed to cry myself to sleep knowing I helped enable this

In all seriousness though I did pass on the cats when making languages because I thought they were just modified humans.

**Comment by Cartlord** [comment]
...Last I knew, they WERE canonically modified humans.

**Comment by JamieD1** [comment]
oh christ.

**Comment by adamsong** [comment]
> ok now give humans a race specific language

Japanese

File: code/modules/language/felinid.dm
Comment (patpol4) [inline code/modules/language/felinid.dm:9]: incredible

Comment (ToasterBiome) [reply code/modules/language/felinid.dm:9]: Is there a problem with this?

**Review by nesisknesis — COMMENTED** [review]
Everyone out here non-chalant about catgirls becoming less of a joke feature and more a genuine concept

what the hell are you all thinking

**Comment by Sniblet** [comment]
As soon as this merges I'm making Karen sheltered

**Comment by JamieD1** [comment]
This just feels so wrong.

**Comment by SomeguyManperson** [comment]
Since you can get it with kitty ears its effectively similar as stoner language which you gain from having drugginess and shouldnt negatively impact the game aside from being cringe
But we already have cringe

**Pull request merged**
